src/components/EditModal.js

```diff
@@ -11,8 +40,20 @@ const EditModal = ({ setToggleEditModal }) => {
           >
             <span className="material-icons-outlined ">close</span>
           </button>
-          <div className="w-36 h-36 bg-primary-200 rounded-full flex justify-center items-center font-bold text-primary-900 absolute  border-solid border-white border-4 text-4xl top-1/4 left-5">
+          <div className="w-36 h-36 bg-primary-200 rounded-full  flex justify-center items-center font-bold text-primary-900 absolute  border-solid border-white border-4 text-4xl top-1/4 left-5">
             T{/* <img src={profileBg} alt="profile-bg" /> */}
+            <label htmlFor="imageFile" className="cursor-pointer">
+              <span className="material-icons-outlined absolute z-20 opacity-5 hover:opacity-80 text-6xl">
+                photo_camera
+              </span>
+            </label>
+            <input
+              type="file"
+              id="imageFile"
+              accept=".png,.jpeg,.jpg"
```

Comment:
you can add these formats also `image/apng, image/avif, image/gif, image/jpeg, image/png, image/svg+xml, image/jpg,image/webp`

Reply:
sure